.style.yapf

```diff
@@ -1,5 +1,5 @@
 [style]
-based_on_style = chromium
+based_on_style = google
```

Comment:
Let's leave this as is for now. I don't want to pollute blame history with formatting changes.

Reply:
Hmm, but this is blocking this PR https://github.com/google/clusterfuzz/pull/4707. The current version doesn't support the `match case` usage. However I can modify the PR to use `if else` instead, and avoid this bump. WDYT?

Reply:
Chromium doesn't support `match case`? Bizzare.
I guess land this then.

Reply:
It doesn't in the version 0.22 ;/. The version 0.29.1 supported but the lint behavior is different as well.